linter.py

```diff
@@ -20,28 +20,12 @@
 class Flow(NodeLinter):
     """Provides an interface to flow."""
 
-    inline_settings = ('coverage', 'all')
-    syntax = (
```

Comment:
syntax *can* be replaced by 'selector' below. This is currently optional, just deprecated.